[GitHub]_ABCDs_Detector.ipynb

```diff
@@ -182,47 +171,49 @@
         "   * The **Run ABCDs Detector** step produces the video analysis.\n",
         "\n",
         "1. For questions, please reach out to: abcds-detector@google.com"
-      ],
-      "metadata": {
-        "id": "XHqQskKjxtEk"
-      }
+      ]
     },
     {
       "cell_type": "markdown",
-      "source": [
-        "# Install ABCD Detector Code"
-      ],
       "metadata": {
         "id": "m-VVH3V36y8y"
-      }
+      },
+      "source": [
+        "# Install ABCD Detector Code"
+      ]
     },
     {
       "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "3KlVPryttrZr"
+      },
+      "outputs": [],
       "source": [
         "!git clone -b paul_changes https://github.com/google-marketing-solutions/abcds-detector\n",
         "!python3 -m pip install -r \"abcds-detector/requirements.txt\"\n",
         "%cd /content/abcds-detector\n",
         "\n",
         "from configuration import Configuration\n",
         "config = Configuration()"
```

Comment:
Move to own section to make the configuration more explicit.  Users running Install may not realize Configuration is also being set. The following code into the notebook will work:
```
    {   
      "cell_type": "markdown",
      "source": [
        "# Install ABCD Detector Code"
      ],
      "metadata": {
        "id": "m-VVH3V36y8y"
      } 
    },  
    {   
      "cell_type": "code",
      "source": [ 
        "!git clone https://github.com/google-marketing-solutions/abcds-detector\n",
        "!python3 -m pip install -r \"abcds-detector/requirements.txt\""
      ],
      "metadata": {
        "id": "3KlVPryttrZr"
      },
      "execution_count": null,
      "outputs": []
    },  
    {   
      "cell_type": "markdown",
      "source": [
        "# Initialize ABCDs Detector Configuration"
      ],
      "metadata": {
        "id": "m-VVH3W36y8y"
      }
    },
    {   
      "cell_type": "code",
      "source": [ 
        "%cd /content/abcds-detector\n",
        "from configuration import Configuration\n",
        "config = Configuration()"
      ],
      "metadata": {
        "id": "3KlVPrWttrZr"
      },
      "execution_count": null,
      "outputs": []
    },
```

Reply:
The colab didn't have any changes other than a method name change. I would not make more changes to this unless something is actually broken.